ui/src/components/References/GroupReference.tsx

```diff
@@ -60,7 +60,7 @@ export default function GroupReference({
   return (
     <div
       className={cn(
-        'not-prose relative flex items-center rounded-lg  text-base transition-colors hover:border-gray-100 hover:bg-white group-one-hover:border-gray-100 group-one-hover:bg-white',
+        'not-prose relative flex max-w-[250px] items-center rounded-lg bg-white text-base transition-colors hover:border-gray-100 hover:bg-white group-one-hover:border-gray-100 group-one-hover:bg-white',
```

Comment:
would 600 better match our other max widths for chat stuff? can't remember, but it does look a little narrow

Reply:
Why a max-width at all?

Reply:
looks bad at full-width imo

Reply:
Yeah, agreed, looks bad at full width. I feel like 600 is also too huge.

Reply:
I'll add that this width makes these look the same when rendered in both gallery items and chat.